### ISSUE TITLE
irdebug.h missing ?

### ISSUE THREAD

**Issue by stanleyseow** [makerworkshop/arduino_irdump_example#1]
I tried to use the https://github.com/makerworkshop/arduino_apple_remote_example but it uses the rcdebug.h in the apple-remote.h ..

Where can I find the irdebug.h or an updated libraries..

Thanks

**Comment by xiam** [comment]
Yeah, sorry. I started to move things around a lot and I was precisely working on updating the Apple Remote example that still depends on an old version of the library. I'll need more time to stabilize that but if you want you try it you can look here: https://github.com/makerworkshop/arduino_irdump_example/tree/4664932897a9c557083677b4625a42b8d6c777a5

**Comment by stanleyseow** [comment]
Thanks for the prompt response ... I tried to compile it.. but got lots of compilation errors...

So I switch to using IRremote.h libs and print out the button pressed for matching..
http://www.forefront.io/a/arduino_apple_remote_controlled_buzzer
